myInsta/myInsta/delegates/AppDelegate.m

```diff
@@ -15,25 +16,32 @@ @interface AppDelegate ()
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(  NSDictionary *)launchOptions {
-
-    ParseClientConfiguration *config = [ParseClientConfiguration  configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
-
-        configuration.applicationId = @"ggKQqFWf3UOSxwNTz02Diuh4zd7sApTVnO5Tm4cB";
-        configuration.clientKey = @"pAUrKsMvsqUNsm14ylpGbJZUDK7b1NUSd68gF0c0";
-        configuration.server = @"https://parseapi.back4app.com";
+    // Creates connection to the database
+    ParseClientConfiguration *config =
+        [ParseClientConfiguration  configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
+        
+        NSString *path = [[NSBundle mainBundle] pathForResource:keys ofType:plist];
+        NSDictionary *dict = [NSDictionary dictionaryWithContentsOfFile: path];
+        NSString *key = [dict objectForKey:appID];
+        NSString *secret = [dict objectForKey:clientKey];
+        
+        configuration.applicationId = key;
+        configuration.clientKey = secret;
+        configuration.server = serverLink;
     }];
 
     [Parse initializeWithConfiguration:config];
     
-    PFObject *profile = [PFObject objectWithClassName:@"profile"];
-    profile[@"username"] = @"as123";
-    profile[@"screenName"] = @"Abbi Shilts";
-    profile[@"followers"] = @500;
+    // builds an object
+    PFObject *profile = [PFObject objectWithClassName:profileStr];
+    profile[usrnameStr] = fillerusr;
+    profile[screennameStr] = namefiller;
+    profile[followersStr] = @500;
     [profile saveInBackgroundWithBlock:^(BOOL succeeded, NSError *error) {
         if (succeeded) {
-            NSLog(@"Object saved!");
+            NSLog(obSave);
```

Comment:
nit: You could change it to "objectSaved", remember to use self-explanatory names so new folks can understand quickly